test/unit/test_helpers_test.lua

```diff
@@ -46,6 +46,7 @@ g.after_all = function()
 end
 
 function g.test_cluster_helper()
+    t.skip('Deprecated')
```

Comment:
It's only temporarily. For the sake of compatibility we'll need to support uploading oldstyle configs